tests/test_timerange.py

```diff
@@ -99,6 +103,18 @@ def test_bounded_on_right_inclusive(self):
         self.assertEqual(rng.to_sec_nsec_range(), "_1530711653:999999999]")
         self.assertEqual(str(rng), "_1530711653:999999999]")
 
+    def test_bounded_on_right_inclusive__negative(self):
+        rng = TimeRange.from_end(Timestamp(1530711653, 999999999, -1))
+
+        self.assertIn(Timestamp(1530711654, 0, -1), rng)
+        self.assertIn(Timestamp(1530711653, 999999999, -1), rng)
+        self.assertNotIn(Timestamp(), rng)
```

Comment:
Nit: Worth testing a positive timestamp, as below?
```suggestion
        self.assertNotIn(Timestamp(), rng)
        self.assertNotIn(Timestamp(326246400, 0), rng)
```